rpcs3/Emu/VFS.cpp

```diff
@@ -21,7 +21,9 @@ bool vfs::mount(const std::string& dev_name, const std::string& path)
 
 	writer_lock lock(table->mutex);
 
-	return table->mounted.emplace(dev_name, path).second;
+	table->mounted[dev_name] = path;
```

Comment:
oh so that's where we clean it. makes sense now. still does not fix ani's problem though :)